operators/tokenizer/bpe_utils.hpp

```diff
@@ -512,87 +512,87 @@ class TokenWithRegularExp {
 
   // Determine ufal::unilib::unicode regex category given string code.
   static ufal::unilib::unicode::category_t StringToCategory(const std::string & category = ""){
-    ufal::unilib::unicode::category_t cat;
-
     // Since C++ is not an interpreted language, we cannot simply convert the category to an object by typing
     // part of code into a string, so we manually parse it. Note that C++ also does not have switch-case statements.
```

Comment:
"Note that C++ also does not have switch-case statements."

nit: this is not true. see https://en.cppreference.com/w/cpp/language/switch. it just doesn't support string comparison.

Reply:
oh ok good to know, cheers!